tests/api_data/get_endpoint_data.json

```diff
@@ -52,14 +52,27 @@
       }
```

Comment:
I seen @Adam-W1 raise a PR removing this on another repo - might be worth just looking out for potential conflicts if something similar happens to this repo.